src/main/docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM openjdk:8-jdk
+LABEL maintainer="Atomist, Inc. <oss@atomist.com>" \
+      org.label-schema.vendor="Atomist, Inc." \
+      org.label-schema.license="Commercial" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.name="rug-cli" \
+      org.label-schema.url="https://github.com/atomist/rug-cli" \
+      org.label-schema.name="${version}" \
+      org.label-schema.description="${description}" \
+      org.label-schema.build-date="${timestamp}"
+
+# we don't want to run the CLI as root
+RUN groupadd -r atomist && \
+    useradd --home-dir=/home/atomist --create-home -r -g atomist atomist && \
+    mkdir -p /home/atomist/.atomist/repository && \
+    mkdir -p /home/atomist/project
+ENV PATH="$PATH:/home/atomist/rug-cli-${version}/bin"
+ADD rug-cli-${version}-bin.tar.gz /home/atomist
+RUN chown -R atomist.atomist /home/atomist
+WORKDIR /home/atomist/project
+USER atomist
+ENTRYPOINT ["rug"]
```

Comment:
Won't this just run `rug`, which without arguments will print some instructions and exit? For an interactive case like this, isn't it better to avoid `ENTRYPOINT`, rather use `CMD` and provide it with something like:

```
CMD [ "rug", "shell" ]
```